CodeStarterCamp_Week2/Lotto_Step2.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Lotto_Step2.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by Sebin Kwon on 5/8/24.
+//
+
+import Foundation
+
+let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+
+func makeLottoNumbers() -> Set<Int> {
+    var winningLottoSet = Set<Int>()
+    while winningLottoSet.count < 6 {
+        winningLottoSet.insert(Int.random(in: 1...45))
+    }
+    return winningLottoSet
+}
+
+func checkLottoNumbers(winning: Set<Int>) {
+    let myLottoSet = Set(myLottoNumbers)
+    let matchingNumbers: Set<Int> = myLottoSet.intersection(winning)
+    if matchingNumbers.count > 0 {
+    let matchingString = matchingNumbers.map{ String($0) }.joined(separator: ", ")
```

Comment:
```suggestion
    let matchingString = matchingNumbers.map { String($0) }.joined(separator: ", ")
```

- 클로져를 사용할 때 리턴타입, 파라미터 등을 생략하거나 후행 클로져의 사용 등에 대해서 공부해보셔도 좋을 것 같습니다!

Reply:
넵 감사합니다!